apps/stats-mainnet-e2e/cypress.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
chore (blocking): make cypress tests run on a port that is not going to be used by other tests. Currently in use:
* 3000
* 4200

Reply:
You can compare this to either BE or tradiung, one of them is using non-default ports for the config